pinot-common/src/main/java/org/apache/pinot/common/messages/SegmentReloadMessage.java

```diff
@@ -34,19 +35,25 @@ public class SegmentReloadMessage extends Message {
   public static final String RELOAD_SEGMENT_MSG_SUB_TYPE = "RELOAD_SEGMENT";
 
   private static final String FORCE_DOWNLOAD_KEY = "forceDownload";
+  private static final String SEGMENT_LIST_KEY = "segmentList";
 
-  public SegmentReloadMessage(@Nonnull String tableNameWithType, @Nullable String segmentName, boolean forceDownload) {
+  public SegmentReloadMessage(@Nonnull String tableNameWithType, @Nullable List<String> segmentNames,
```

Comment:
(minor, convention) Let's remove the `@Nonnull` annotation as it is very easy to misread as `@Nullable`